api/proto/teleport/discoveryconfig/v1/discoveryconfig.proto

```diff
@@ -80,3 +83,41 @@ enum DiscoveryConfigState {
   // DISCOVERY_CONFIG_STATE_SYNCING is used when the discovery process has started but didn't finished yet.
   DISCOVERY_CONFIG_STATE_SYNCING = 3;
 }
+
+// AWSEC2InstancesDiscovered represents an AWS EC2 Instance that was discovered.
+message AWSEC2InstancesDiscovered {
+  // Region is the AWS Region.
+  string region = 1;
+  // Integration is the AWS OIDC Integration name used to fetch the resource.
+  string integration = 2;
+  // InstanceID is the instance ID which uniquely identifies this resource in AWS.
+  string instance_id = 3;
+  // EnrollMode indicates the enroll mode used to auto-enroll this instance.
+  // Either using the EICE or the Script mode.
+  types.InstallParamEnrollMode enroll_mode = 4;
+  // EnrollStatus indicates whether the instance was successfully enrolled or if it failed.
+  AWSEC2EnrollmentStatus enroll_status = 5;
+  // EnrollMessage is an informational message used to further expand on the result of auto-enrolling this instance.
+  string enroll_message = 6;
+  // Name is the friendly name of the resource.
+  // Might be empty.
+  string name = 7;
+  // SSMInvocationURL is the URL that points to the ssm:RunCommand invocation result.
+  // This only present when auto-enrolling using the Script mode.
```

Comment:
Is it expected to be a URL to an S3 resource? Also: does "result" mean "output", as in stdout, or just an exit code?

Reply:
The link will look like this
https://eu-west-2.console.aws.amazon.com/systems-manager/run-command/ff3fe991-1535-4a2e-b26f-bb526a3c4b41/i-0c373df52f7c094c5?region=eu-west-2
It contains the result of invoking an SSM Document to an EC2 instance.
It contains all the executed steps (eg, download teleport script, run teleport script), and for each step the exit code and the stdout/stderr.
(you can click on the link above to see an example).